_includes/structure/javascript.html

```diff
@@ -13,8 +13,10 @@
 <!-- Account Registration modifications -->
 <script src="{{ site.url }}{{ site.baseurl }}/js/register.js"></script>
 
-
 <!-- Cookie management -->
+<script src="https://cdn.jsdelivr.net/gh/orestbida/cookieconsent@v2.9.2/dist/cookieconsent.js"></script>
```

Comment:
This needs changing to include an integrity sha (like the jQuery script loaded at the top of the file).

If we don't do this, then it leaves the site open to potential security vulnerabilities in the future. It is likely this would be flagged in a PEN test.

For more info see:

https://cheatsheetseries.owasp.org/cheatsheets/Third_Party_Javascript_Management_Cheat_Sheet.html#subresource-integrity

MDN has an example showing you to generate the appropritate integrity sha hash here:

https://developer.mozilla.org/en-US/docs/Web/Security/Practical_implementation_guides/SRI#examples

Reply:
Added and added for earlier JS cdn for popper.